.github/workflows/roundtrip/docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 services:
   keycloak:
     # This is kc 24.0.1 with opentdf protocol mapper on board
```

Comment:
Can you remove the stale comment on line 3 please?

Reply:
good catch!